connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceTask.java

```diff
@@ -106,13 +107,28 @@ public void start(Map<String, String> props) {
         Map<TopicPartition, Long> topicPartitionOffsets = loadOffsets(taskTopicPartitions);
         consumer.assign(topicPartitionOffsets.keySet());
         log.info("Starting with {} previously uncommitted partitions.", topicPartitionOffsets.entrySet().stream()
-            .filter(x -> x.getValue() == 0L).count());
-        log.trace("Seeking offsets: {}", topicPartitionOffsets);
-        topicPartitionOffsets.forEach(consumer::seek);
+            .filter(x -> x.getValue() == NON_EXISTING_OFFSET_VALUE).count());
```

Comment:
Nit: we can clean this up if we use the `isUncommitted` method

```suggestion
        log.info("Starting with {} previously uncommitted partitions.", topicPartitionOffsets.values().stream()
                .filter(this::isUncommitted).count());
```